### PR TITLE
fix(maxmsp): wrong and missing function signture

### PR DESCRIPTION
**Pull request by FrancescElies**
Updates two functions and adds a missing one, see inline code comments

### CODE REVIEW

File: types/maxmsp/index.d.ts [hunk 1]
```diff
@@ -28,7 +28,10 @@ declare var patcher: Patcher;
 declare function error(message: any): void;
 declare function cpost(message?: any): void;
 declare function post(message?: any): void;
-declare function messnamed(object_name: string, message_name: string, message_arguments?: string): void;
+/**
+ * Sends a message to the named Max object. A named Max object is an object associated with a global symbol (not an object with a patcher-specific name). For example, Max receive objects are bound to global symbols. The following code would send the message bang to the named object flower.
+ */
+declare function messnamed(object_name: string, message_name: string, message_arguments?: string | number): void;
```
Comment (FrancescElies) [inline types/maxmsp/index.d.ts:34]: message_arguments accepts a number too

File: types/maxmsp/index.d.ts [hunk 2]
```diff
@@ -1209,10 +1212,11 @@ declare class Maxobj {
...
 
     /**
-     * If the object contains a patcher, this function returns a (Javascript) Patcher object. The optional index is used for specifying an instance number, which only applies to poly~ objects. If the
-     * object does not contain a subpatcher, a nil value is returned.
+     * If the object contains a patcher, this function returns a (Javascript) Patcher object.
+     * The optional index is used for specifying an instance number, which only applies to poly~ objects.
+     * If the object does not contain a subpatcher, a nil value is returned.
      */
-    subpatcher(index: number): Patcher;
+    subpatcher(index?: number): Patcher;
```
Comment (FrancescElies) [inline types/maxmsp/index.d.ts:1219]: index is optoinal

File: types/maxmsp/index.d.ts [hunk 3]
```diff
@@ -1297,6 +1301,13 @@ declare class Patcher {
      */
     wind: Wind;
 
+    /**
+     * Sends message to the patcher followed by any additional arguments (..anything) provided.
+     * This is useful for manupulating patchers which dynamically can do things like: creating new buttons,
+     * resizing it's window, switching to presentation mode, ...
+     */
+    message(message: string, ...anything: any[]): void;
```
Comment (FrancescElies) [inline types/maxmsp/index.d.ts:1309]: one can send messages to a Patcher too.

**Review by twhiston — APPROVED** [review]

**Comment by FrancescElies** [comment]
Ready to merge

**Pull request merged**